### ISSUE TITLE
[Feature Request] Messages.yml Enhancements

### ISSUE THREAD

**Issue by MeMyselfMC** [SpigotMC/BungeeCord#1214]
Upon starting work on my network, I noticed something that bugs me a little bit with BungeeCord. The messages.yml file is kinda incomplete.

By incomplete, I mean something like this:

current_server=\u00a76You are currently connected to

When it could look like this, and the {0} variable would be replacing the server the player is currently on:

current_server=\u00a76You are currently connected to {0}!

Basically, what the messages.yml file needs is more variables. That would allow us to better customize messages, and remove the errors which players won't understand if they come across one in a kick/connect messages.
